spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher/response_yamls/external_network_providers.yml

```diff
@@ -6,11 +6,11 @@ internal:
   description: 
   id: 6e452166-e772-4355-b050-24705a32c15c
   name: ovn
-  authentication_url: http://localhost:35357/v2.0/
+  authentication_url: http://bodnopoz-engine.eng.lab.tlv.redhat.com:35357/v2.0/
```

Comment:
Can you change these back to localhost?  Best to not have internal DNS names in public spec helpers.

Reply:
done